webapp/src/containers/Rhs/rhsData.tsx

```diff
@@ -67,6 +74,10 @@ const RhsData = ({
         list: subscription.subscription_events.split(',').map((event) => SubscriptionEventLabels[event as SubscriptionEvents]),
     }), []);
 
+    const hasMoreSubscriptions = useMemo<boolean>(() => (
+        subscriptionList.length !== 0 && (subscriptionList.length - (paginationQueryParams.page * Constants.DefaultPageSize) === Constants.DefaultPageSize)
```

Comment:
```suggestion
        subscriptionList.length > 0 && (subscriptionList.length - (paginationQueryParams.page * Constants.DefaultPageSize) === Constants.DefaultPageSize)
```
Can you explain this logic? I think it's wrong. :thinking:

Reply:
updated the logic now